docs/rules/spaced-comment.md

```diff
@@ -175,6 +175,12 @@ Examples of **incorrect** code for this rule with the `"always"` option combined
 /*-+-+-+-+-+-+-+*/
 ```
 
+```js
+/* eslint spaced-comment: ["error", "always", { "line": { "exceptions": ["*"] } }] */
```

Comment:
With this configuration, the exception does not apply to the comment below just because it is a `Block` comment.

I think this example would provide more explanation about the behavior of this rule if we change `"*"` to be an exception for `"block"` comments (or simply all comments), not `"line"` comments.

Reply:
@mdjermanovic like this 

- incorrect

```js
/* eslint spaced-comment: ["error", "always", { "block": { "exceptions": ["*"] }}] */

/**** COMMENT
*
*/
```

- correct

```js
/* eslint spaced-comment: ["error", "always", { "block": { "exceptions": ["*"] }}] */

/****
*
*/
```

Reply:
The examples you added already look great, I think we should just change the config, from:

`/* eslint spaced-comment: ["error", "always", { "line": { "exceptions": ["*"] } }] */`

to:

`/* eslint spaced-comment: ["error", "always", { "block": { "exceptions": ["*"] } }] */`

in order to make it clear that `/******** COMMENT *******/` doesn't match the exception pattern.

Reply:
done 👍